views/install.sh.erb

```diff
@@ -169,6 +169,14 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
```

Comment:
I did not find any.  I can change this to do some grep / split work to grab just what we want out of the file if you think it would be better.  I was just taking advantage of the file's already-parseable-by-bourne nature.